### ISSUE TITLE
Feature request: Per node/edge attributes

### ISSUE THREAD

**Issue by josqu4red** [mingrammer/diagrams#21]
Great library!

It would be awesome to support per-resource attributes, to highlight relations between different sub-systems, for instance.

**Comment by mingrammer** [comment]
It would be good to support them. But I have no idea how to support `per edge` attributes. Per node attribute is very easy, `EC2('name', attr=...)` for instance. But the edge only can be made by shift operator now, so there is no place to inject the attribute.

Any idea?

**Comment by warmfusion** [comment]
Could you do something where if the shifted target is a string treat it as a edge label?

```
EC2("MyNode") >> "EdgeNote" >> EC2("OtherNode")
```

For more advanced attributes, a formal object could be used?

```
attr = Attribute(dashed=True, color="pantone-448-c", label="Goes here")
node >> attr >> other_node
```

**Comment by ki11roy** [comment]
Please take a look at #48

**Comment by mingrammer** [comment]
#48 merged and Diagrams v0.7.0 is now released.
